components/style/properties.mako.rs

```diff
@@ -1624,7 +1664,12 @@ pub mod longhands {
 
     // CSS 2.1, Section 15 - Fonts
 
-    ${new_style_struct("Font", is_inherited=True)}
+    ${new_style_struct("Font", is_inherited=True, gecko_name="nsStyleFont",
```

Comment:
Aaaannnddd... There it goes: https://github.com/servo/servo/pull/10276

Turns out the dumb way I was generating untranslatable fields not always worked.